css/store.css

```diff
@@ -0,0 +1,289 @@
+header {
+  height: 100px;
+  display: flex;
+  position: relative;
+}
+header :nth-child(2) {
+  margin-left: auto;
+  gap: 50px;
+  margin-right: 30px;
+}
+.burger-button {
+  display: none;
+  font-size: 30px;
+  cursor: pointer;
+  margin-left: auto;
+  text-decoration: none;
+}
+header img {
+  width: 70px;
+  height: 50px;
+  padding: 15px;
+  margin-left: 3rem;
+}
+.nav-header {
+  display: flex;
+  gap: 10px;
+  padding: 15px;
+  align-items: center;
+}
+a {
+  text-decoration: none;
+  color: rgb(129, 121, 121);
+}
+
+a:hover {
+  color: brown;
+  font-weight: bold;
+}
+
+.nav-header :nth-child(1) {
+  font-weight: bold;
+  color: rgb(85, 81, 81);
+}
+
+.container {
+  display: flex;
+}
+.element1 {
+  width: 50%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.element2 {
+  width: 50%;
+  height: 100%;
+}
+.element2 img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+h1 {
+  text-align: center;
+  font-size: 50px;
+  padding: 30px;
+  margin-top: 3rem;
+  color: rgba(240, 81, 18, 0.904);
+}
+form {
+  width: 80%;
+  height: auto;
+}
+.form {
+  margin-top: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  margin-bottom: 1rem;
+}
+label {
+  display: block;
+  margin-bottom: 10px;
+  font-weight: bold;
+}
+.form-address {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+input[type="text"],
+select {
+  width: 100%;
+  padding: 12px;
+  border: 1px solid #ccc;
+  border-radius: 10px;
+  box-sizing: border-box;
+}
+.address11 {
```

Comment:
Continuing with the address example.
Let's say you have 2 address fields. The first is required and has to be filled, the second is optional.
Instead of creating classes "address1" and "address2", a better way is to create classes "address" and "required".
This way if you later need a third address field, you don't need to edit your css. You will just reuse the "address" class.
Also maybe you will have other required fields having some similarities. Then the "required" class can be reused as well.

Reply:
A bit more on the "required" class for address.
You could add "*" in css. It would be something like this:
```
.required::after {
  content: '*';
}
```
Haven't tested this code. Please tell if you will try and it doesn't work. Can look into it.

Reply:
"::after" above is a so called "pseudoelement" You can read more in here for example https://developer.mozilla.org/en-US/docs/Web/CSS/::after

Reply:
"Yes, it's a good approach to use just one class instead of multiple. I completely agree with you, and I'll work on resolving it. Thank you so much for your support!"

Reply:
> A bit more on the "required" class for address. You could add "*" in css. It would be something like this:
> 
> ```
> .required::after {
>   content: '*';
> }
> ```
> 
> Haven't tested this code. Please tell if you will try and it doesn't work. Can look into it.

"I used the .quote class to automatically add quotation marks before and after the content, and it works well. I’ve tested it, and everything seems fine."